src/main/resources/application.yaml

```diff
@@ -45,6 +45,43 @@ idam:
       username: ${IDAM_USERS_BULKSCAN_USERNAME:bulkscan+ccd@gmail.com}
       password: ${IDAM_USERS_BULKSCAN_PASSWORD:Password12}
 
+service-config:
+  # Services and all their case type IDs.
+  # These mappings are used by Bulk Scan to determine which case type IDs to search across
+  # when looking for a case by its legacy ID. Those CTIDs that are not present here, will
+  # not be included in the search.
+  services:
+    - service: bulkscan # service name corresponds to the name of the container in Blob Storage
+      jurisdiction: BULKSCAN
+      case-type-ids:
+        - Bulk_Scanned
+        - BULKSCAN_ExceptionRecord
+
+    - service: sscs
+      jurisdiction: SSCS
+      case-type-ids:
+        - SSCS_ExceptionRecord
```

Comment:
I think we should not configure exception record case types ? We should only have service case types as we are attaching scanned documents to service cases.

Reply:
Of course. They won't even have legacy IDs, anyway. Will remove.